oar/core/const.py

```diff
@@ -20,6 +20,10 @@
 LABEL_BUG_FIRST_CELL = "C8"
 LABEL_ISSUES_OTHERS_COLUMN = "H"
 LABEL_ISSUES_OTHERS_ROW = 8
+LABEL_START_ROW_INDEX = 1
+LABEL_END_ROW_INDEX = 2
+LABEL_START_COL_INDEX = 1
+LABEL_END_COL_INDEX = 2
```

Comment:
It should be clear from the name that it is advisory info cell.
For example  `LABEL_ADVISORY_START_ROW_INDEX`